src/crawlee/http_clients/_base.py

```diff
@@ -52,6 +55,8 @@ async def crawl(
     ) -> HttpCrawlingResult:
         """Perform the crawling for a given request.
 
+        This method is called from `crawler.run()`.
```

Comment:
Huh, who is this for? I think that a developer should be able to just search for references, and if we change this, we'd need to update this doctring.

Plus I believe that we'll manage to simplify the BaseHttpClient interface anyway.

Reply:
Intended for anyone who implements their own HTTP client. I was asking this question myself as well. Maybe we can leave it there until we improve & simplify it?

Reply:
That works for me.